src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/legacy/model/NomisTermType.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.legacy.model
+
+enum class NomisTermType(var description: String = "") {
```

Comment:
Is Legacy* another way of saying Nomis*?? in which case can this be called LegacyTermType?